### ISSUE TITLE
add link titles for event cards

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/cdh-web#167]
there's no accessible name for these links currently:
```html
<a property="schema:url" href="/events/2019/03/unsolved-data-problems/">
...
</a>
```

**Comment by thatbudakguy** [comment]
check in VO and it will read the full card text when focused currently, which I think is sufficient. it's a little more text than is necessary, but probably preferable to repeating a bunch of content when you focus the card, and you can always tab away when you hear enough.
